apps/fz_wall/lib/fz_wall/server.ex

```diff
@@ -15,95 +15,116 @@ defmodule FzWall.Server do
 
   @impl GenServer
   def init(_rules) do
-    cli().teardown_table()
-    cli().setup_table()
-    cli().setup_chains()
-    {:ok, existing_rules} = GenServer.call(http_pid(), :load_rules, @init_timeout)
-    cli().restore(existing_rules)
-    {:ok, existing_rules}
+    cli().setup_firewall()
+    {:ok, settings} = GenServer.call(http_pid(), :load_settings, @init_timeout)
+    cli().restore(settings)
+    {:ok, settings}
```

Comment:
It's better to have a map as GenServer state when there are multiple items.

If we change settings to `%{rules: ..., devices: ..., users: ...}`,

We can change the handle_* functions to be like:

```elixir
def handle_call({:add_rule, rule}, _from, %{rules: existing_rules} = state) do
  new_rules = add_rule(rule, existing_rules)
  {:reply, :ok, %{state | rules: new_rules}}
end
```

`%{map | key: value}` is a special update syntax for existing keys only.